src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyView.js

```diff
@@ -7,7 +7,42 @@ import { Body as SelectAuthenticatorVerifyViewBody } from '../SelectAuthenticato
 import { AUTHENTICATOR_KEY } from '../../../ion/RemediationConstants';
 
 const Body = SelectAuthenticatorVerifyViewBody.extend({
```

Comment:
As per suggestion, I will be creating a separate view for the 'Send Push' use case. That view will extend from BaseForm directly. I am working on it will update the PR soon.